setup/providers/protonvpn.md

```diff
@@ -46,10 +44,12 @@ services:
 - `OPENVPN_USER` is your **OPENVPN specific** username. Find it at [account.proton.me/u/0/vpn/OpenVpnIKEv2](https://account.proton.me/u/0/vpn/OpenVpnIKEv2).
 - `OPENVPN_PASSWORD`
 
-### Wireguard only
+### WireGuard only
 
 - `VPN_TYPE=wireguard`
-- `WIREGUARD_PRIVATE_KEY` is your 32 bytes key in base64 format. The private key can only be obtained by [generating a Wireguard configuration file](https://account.protonvpn.com/downloads). Generate a Wireguard configuration file, copy the displayed `PrivateKey` value and optionally download the configuration file. Note this value is the same for all ProtonVPN servers. 💁 [Guide on how to generate a configuration file](https://protonvpn.com/support/wireguard-configurations/)
+- `WIREGUARD_PRIVATE_KEY` is your 32 byte key in base64 format.
+
+💁 To obtain the `WIREGUARD_PRIVATE_KEY` value, provision a configuration file from [account.proton.me/u/0/vpn/WireGuard](https://account.proton.me/u/0/vpn/WireGuard). The content of the WireGuard configuration file that ProtonVPN generates contains the `PrivateKey` value in the `[Interface]` section. Note that the private key value is the same for all ProtonVPN servers. All other values in the ProtonVPN WireGuard Configuration file should be ignored and not set in Gluetun environment variables. [ProtonVPN documentation on how to generate a WireGuard configuration file](https://protonvpn.com/support/wireguard-configurations/)
```

Comment:
Only changed the wireguard config file generation link, since the rest of the paragraph is essentially just rewording the same thing imo

```suggestion
- `WIREGUARD_PRIVATE_KEY` is your 32 byte key in base64 format. The private key can be obtained by [generating a Wireguard configuration file](https://account.proton.me/u/0/vpn/WireGuard) and copy the displayed `PrivateKey` value. Note this value is the same for all ProtonVPN servers. 💁 [Guide on how to generate a configuration file](https://protonvpn.com/support/wireguard-configurations/)
```